coralproject/talk
# [CORL-220] Change email address

## What does this PR do?
- allows user to change their own email address provided they correctly enter their password
- sets `emailVerified` to false and sends out a new email verification email
- gives user the option to re-send email verification if their email address is unverified.

## How do I test this PR?
- visit profile tab
- click "edit" next to email 
- fill out a new email address and enter password
- you should see a notice about email verification

Very similar to change username PR, with additional step on the backend of updating the user profile ID for local profiles.
